chain/chaintest/balance_handler_test_helpers.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package chaintest
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/hypersdk/chain"
+	"github.com/ava-labs/hypersdk/codec/codectest"
+	"github.com/ava-labs/hypersdk/state/tstate"
+)
+
+// TestBalanceHandler tests [b] by calling the following methods (in order):
+// - SponsorStateKeys()
+// - AddBalance()
+// - CanDeduct()
```

Comment:
Could we complete this comment with all of the steps this test is performing?

Reply:
Done